src/crate/client/sqlalchemy/doctests/itests.txt

```diff
@@ -65,10 +65,10 @@ Retrieve the location from the database::
     >>> location.name
     'Earth'
 
-Date should have been set at the insert due to default value via python method::
+Date should have been set at the insert due to default value via Python method::
 
-    >>> from datetime import datetime
-    >>> now = datetime.utcnow()
+    >>> from datetime import datetime, timezone
+    >>> now = datetime.now(timezone.utc).replace(tzinfo=None)
```

Comment:
I don't understand the change here.  Isn't `now(timezone.utc).replace(tzinfo=None)` the same as `utcnow()`?

Reply:
It is really sad. In normal situations, or when the system time zone setting is expressed in UTC, everything is fine. However, it is not the case if the system time zone is anything other than UTC.

I've tried to improve the situation to outline the problem with c1f183ed4.

Reply:
This is not the case.

[utcnow](https://docs.python.org/3/library/datetime.html?highlight=datetime#datetime.datetime.utcnow):

> Return the current UTC date and time, with [tzinfo](https://docs.python.org/3/library/datetime.html?highlight=datetime#datetime.datetime.tzinfo) None.

Yes, it is a naive datetime without timezone info attached, but it _is_ using utc.

```
>>> print(datetime.utcnow(), '\n', datetime.now(tz=timezone.utc).replace(tzinfo=None), '\n', datetime.now())
2022-07-22 18:27:27.055192
 2022-07-22 18:27:27.055200
 2022-07-22 20:27:27.055213
```


```
↪  date +"%Z %z"
CEST +0200
```

Reply:
Also as the articles point out, the problem is the use of the naive datetime (lack of tzinfo) in subsequent operations. If we remove the tzinfo anyways via `.replace(tzinfo=None)` we don't gain anything by using `now(tz=timezone.utc)` but only make things more complicated.

Reply:
Thank you. You convinced me that this goes too far. I've submitted #441 instead, which solely focuses on mitigating a fluke I ~~regularly~~ occasionally observed when running the test suite on CI after midnight.